src/Embeddings/VectorStores/Elasticsearch/ElasticsearchVectorStore.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace LLPhant\Embeddings\VectorStores\Elasticsearch;
+
+use Elastic\Elasticsearch\Client;
+use Elastic\Elasticsearch\Exception\ClientResponseException;
+use Elastic\Elasticsearch\Exception\MissingParameterException;
+use Elastic\Elasticsearch\Exception\ServerResponseException;
+use LLPhant\Embeddings\Document;
+use LLPhant\Embeddings\VectorStores\VectorStoreBase;
+
+class ElasticsearchVectorStore extends VectorStoreBase
+{
+    final public const INDEX_NAME = 'llphant';
```

Comment:
you should make it extendable

Reply:
I don't get it, you can call a parent const on a children of a class no ? (ElasticsearchVectorStoreChild::INDEX_NAME)

Reply:
INDEX_NAME is only used as default value, we can set the $indexName in the constructor so it's good